core/src/scene/styleContext.cpp

```diff
@@ -361,6 +361,14 @@ void StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
             case StyleParamKey::text_source_right:
                 _val = value;
                 break;
+            case StyleParamKey::size: {
```

Comment:
Tangram JS doesn't seem to support returning size strings from parameter functions, so we should keep that consistent.

Reply:
This is to handle `auto` and `%`. I need to check if this use case is supported on JS via functions.

Reply:
If not just simplifies things for us.

Reply:
Looks like it doesn't, based on http://tangram.city/play

Reply:
thanks will take it out.

Reply:
addressed. Thanks